docs_src/OVMS/profileLauncherConfigs.md

```diff
@@ -0,0 +1,63 @@
+# Profile Configuration
+
+For the profile launcher, each profile has its own configuration for different pipelines.  The configuration of each profile is done through an yaml configuration file, configuration.yaml.  One example of configuration.yaml is [shown here for classification](https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/cmd_client/res/classification/configuration.yaml):
+
+```yaml
+OvmsSingleContainer: false
+OvmsServer:
+  ServerDockerScript: start_ovms_server.sh
+  ServerDockerImage: openvino/model_server:2023.1-gpu
+  ServerContainerName: ovms-server
+  ServerConfig: "/models/config.json"
+  StartupMessage: Starting OVMS server
+  InitWaitTime: 10s
+  EnvironmentVariableFiles:
+    - ovms_server.env
+  # StartUpPolicy:
+  # when there is an error on launching ovms server startup, choose one of these values for the behavior of profile-launcher: 
+  #   remove-and-restart: it will remove the existing container with the same container name if any and then restart the container
+  #   exit: it will exit the profile-launcher and 
+  #   ignore: it will ignore the error and continue (this is the default value if not given or none of the above)
+  StartUpPolicy: ignore    
+OvmsClient:
+  DockerLauncher:
+    Script: docker-launcher.sh
+    DockerImage: python-demo:dev
+    ContainerName: classification
+    Volumes:
+      - "$RUN_PATH/results:/tmp/results"
+      - ~/.Xauthority:/home/dlstreamer/.Xauthority
+      - /tmp/.X11-unix
+  PipelineScript: ./classification/python/entrypoint.sh
+  PipelineInputArgs: "" # space delimited like we run the script in command and take those input arguments
+  EnvironmentVariableFiles:
+    - classification.env
+```
+
+The description of each configuration element is explained below:
+
+---
+
+| Configuration Element                   | Description                     |
+| ---------------------------------------------- | ------------------------------------------ |
+| OvmsSingleContainer     | This boolean flag indicates whether this profile is running as a single OpenVino Model Server (OVMS) container or not, e.g. the C-API pipeline use case will use this as `true`. <br>It can indicate the distributed architecture of OVMS client-server when this flag is false. </br>  |
```

Comment:
For consistency should we change Ovms to OVMS everywhere ?

Reply:
sure

Reply:
the configuration element name should not be changed for now as they are implemented that way.  we can have another PR for that kind of change.   for this PR it is documentation; so the configuration element name stays with Ovms. @NeethuES-intel